### ISSUE TITLE
Bogus "Unsupported Java Version: 11" when checking a model

### ISSUE THREAD

**Issue by lemmy** [alygin/vscode-tlaplus#157]
![Capture](https://user-images.githubusercontent.com/88777/84068153-647ad880-a97d-11ea-80b5-559b1e32b5e7.PNG)

**Comment by alygin** [comment]
@lemmy, could you, please, provide the output of `java -version` on your computer? The `java` must be the one configured in the extension. Thanks.

**Comment by lemmy** [comment]
I see what's going on: The extension was configured to use a Java installation installed into Ubuntu running in WSL2 (`/usr/lib/jvm/...`).  However, VSCode was - no longer - running in Remote Workspace mode.  Once I switched to the Java installation installed for Windows, the warning disappeared.

I suggest to include the full qualified Java path in the warning dialog.

**Comment by alygin** [comment]
I think I'll add a button that shows the full path and the command output in such cases, but this message still bothers me. If java was not available at the configured path, I would expect a different message. Here's what the extension shows when it happens:

<img width="384" alt="image" src="https://user-images.githubusercontent.com/2101250/84185414-a4ce7980-aa97-11ea-9ce6-e7bd17d15f67.png">

The only case I would expect the message you provided is when you have Java that only reports its major version when called with the `-version` argument. I've just never seen that before.

**Comment by lemmy** [comment]
```
PS C:\Program Files\Java\jdk-11\bin> .\java -version
java version "11" 2018-09-25
Java(TM) SE Runtime Environment 18.9 (build 11+28)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11+28, mixed mode)
```
`c:\Program Files\Java\jdk-11\release`
```
BUILD_TYPE="commercial"
IMPLEMENTOR="Oracle Corporation"
IMPLEMENTOR_VERSION="18.9"
JAVA_VERSION="11"
JAVA_VERSION_DATE="2018-09-25"
MODULES="java.base java.compiler java.datatransfer java.xml java.prefs java.desktop java.instrument java.logging java.management java.security.sasl java.naming java.rmi java.management.rmi java.net.http java.scripting java.security.jgss java.transaction.xa java.sql java.sql.rowset java.xml.crypto java.se java.smartcardio jdk.accessibility jdk.internal.vm.ci jdk.management jdk.unsupported jdk.internal.vm.compiler jdk.aot jdk.internal.jvmstat jdk.attach jdk.charsets jdk.compiler jdk.crypto.ec jdk.crypto.cryptoki jdk.crypto.mscapi jdk.dynalink jdk.internal.ed jdk.editpad jdk.hotspot.agent jdk.httpserver jdk.internal.le jdk.internal.opt jdk.internal.vm.compiler.management jdk.jartool jdk.javadoc jdk.jcmd jdk.management.agent jdk.jconsole jdk.jdeps jdk.jdwp.agent jdk.jdi jdk.jfr jdk.jlink jdk.jshell jdk.jsobject jdk.jstatd jdk.localedata jdk.management.jfr jdk.naming.dns jdk.naming.rmi jdk.net jdk.pack jdk.rmic jdk.scripting.nashorn jdk.scripting.nashorn.shell jdk.sctp jdk.security.auth jdk.security.jgss jdk.unsupported.desktop jdk.xml.dom jdk.zipfs"
OS_ARCH="x86_64"
OS_NAME="Windows"
SOURCE=".:c89379adc2f4 open:76072a077ee1"
```

**Comment by alygin** [comment]
Thank! So, it really only reports "11" as its version. That explains the message. I'll fix the incorrect parsing too.

**Comment by alygin** [comment]
@lemmy, here's a build with both fixes: [v1.5.1-alpha.1](https://github.com/alygin/vscode-tlaplus/releases/tag/v1.5.1-alpha.1)

**Comment by alygin** [comment]
Fixed in v1.5.1
